packages/medusa-react/src/hooks/admin/publishable-api-keys/mutations.ts

```diff
@@ -0,0 +1,66 @@
+import { useMutation, UseMutationOptions, useQueryClient } from "react-query"
+import { Response } from "@medusajs/medusa-js"
+
+import {
+  AdminPublishableApiKeyDeleteRes,
+  AdminPublishableApiKeysRes,
+} from "@medusajs/medusa"
+
+import { buildOptions } from "../../utils/buildOptions"
+import { useMedusa } from "../../../contexts"
+import { adminPublishableApiKeysKeys } from "."
+
+export const useAdminCreatePublishableApiKey = (
+  options?: UseMutationOptions<Response<AdminPublishableApiKeysRes>, Error, {}>
+) => {
+  const { client } = useMedusa()
+  const queryClient = useQueryClient()
+  return useMutation(
+    (payload: {}) => client.admin.publishableApiKeys.create(payload),
```

Comment:
question: Same as previous, but I ll stop here for this question :)

Reply:
because at the moment we don't pass any data when creating a publishable key, so I left an object for consistency (but this depends on the CRUD PR)